woocommerce/woocommerce-ios
# Dashboard: Optimized SiteVisitStats data fetch with new URL param

Thanks to some great advice from @nbradbury, we are using a new URL param on `SiteVisitStatsRemote` — `stat_fields=visitors`. Doing this makes the endpoint response time _way faster_. 

Soooo this PR:
* Updates the `SiteVisitStatsRemote` to use the new URL param
* Updates `SiteVisitStatsItem` by removing all of the unused params that are no longer returned in the response json
* Updates the sham response bodies in the unit tests
* Updates the unit tests in `SiteVisitStatsMapperTests` accordingly

## Testing

* Build and run the app. 
  * On initial launch, verify you see the visitors data in the dashboard (it should be _much_ quicker)
  * Pull to refresh, verify you see the visitors data in the dashboard (again, it should be quicker)
* Make sure the unit tests are ✅
